Child/dependent care credit should be refundable in 2021

According to the [2021 IRS Publication 503](https://www.irs.gov/pub/irs-pdf/p503.pdf), the CDCC was refundable in 2021.  It does not seem as if PolicyEngineUS incorporates this one-year feature.   But because TAXSIM35 does make the CDCC refundable in 2021, the following test case has a $5000 difference in the US income tax liability.

```
- name: Tax unit with taxsimid 80649 in r21.its.csv and r21.ots.csv
  absolute_error_margin: 0.01
  period: 2021
  input:
    people:
      person1:
        is_tax_unit_head: true
        age: 58
        employment_income: 21_010
        taxable_interest_income: 1_010
        social_security: 23_000
        real_estate_taxes: 14_000
        interest_expense: 22_000
        ssi: 0  # not in TAXSIM35
        state_supplement: 0  # not in TAXSIM35
        wic: 0  # not in TAXSIM35
      person2:
        age: 11
        ssi: 0  # not in TAXSIM35
        state_supplement: 0  # not in TAXSIM35
        wic: 0  # not in TAXSIM35
      person3:
        age: 11
        ssi: 0  # not in TAXSIM35
        state_supplement: 0  # not in TAXSIM35
        wic: 0  # not in TAXSIM35
    spm_units:
      spm_unit:
        members: [person1, person2, person3]
        snap: 0  # not in TAXSIM35
        tanf: 0  # not in TAXSIM35
    tax_units:
      tax_unit:
        members: [person1, person2, person3]
        tax_unit_childcare_expenses: 10_000
        premium_tax_credit: 0  # not in TAXSIM35
    households:
      household:
        members: [person1, person2, person3]
        state_code: TX  # no state income tax
  output:  # expected results from TAXSIM35 12/08/22 version
    taxsim_tfica: 1_607.26
    income_tax: -19_756.34
```

Here is the test result:
```
================================= FAILURES ===================================
_______________________________ test session _________________________________
/Users/mrh/work/Policy-Engine-US/US/r21-id80649.yaml:
  Test 'Tax unit with taxsimid 80649 in r21.its.csv and r21.ots.csv':
    income_tax@2021: [-14756.344] differs from -19756.33984375 
                                  with an absolute margin [4999.996] > 0.01
========================= short test summary info ============================
FAILED r21-id80649.yaml::
============================ 1 failed in 1.53s ===============================
```

The two models agree on AGI, itemized deductions, taxable income (zero), and on the CTC and EITC and RRC amounts.   But TAXSIM35 gives this household a $5,000 refundable CDCC, while PolicyEngineUS does not (evidently because it thinks the CDCC was a non-refundable credit in 2021).

--- a/changelog_entry.yaml
+++ b/changelog_entry.yaml
@@ -1,3 +1,4 @@
 - bump: patch
   changes:
-    - Use parameter to define refundable and non-refundable credits, fixing CDCC refundability under ARPA.
+    fixed:
+      - Use parameter to define refundable and non-refundable credits, fixing CDCC refundability under ARPA.

--- a/changelog_entry.yaml
+++ b/changelog_entry.yaml
@@ -1,0 +1,3 @@
+- bump: patch
+  changes:
+    - Use parameter to define refundable and non-refundable credits, fixing CDCC refundability under ARPA.

--- a/policyengine_us/variables/gov/irs/credits/income_tax_non_refundable_credits.py
+++ b/policyengine_us/variables/gov/irs/credits/income_tax_non_refundable_credits.py
@@ -10,18 +10,7 @@ class income_tax_non_refundable_credits(Variable):
         "Total non-refundable credits used to reduce positive tax liability"
     )
     unit = USD
-    adds = [
-        "cdcc",
-        "elderly_disabled_credit",
-        "non_refundable_ctc",
-        "non_refundable_american_opportunity_credit",
-        "lifetime_learning_credit",
-        "retirement_savings_credit",
-        "residential_clean_energy_credit",
-        "energy_efficient_home_improvement_credit",
-        "new_clean_vehicle_credit",
-        "used_clean_vehicle_credit",
-    ]
+    adds = "gov.irs.credits.non_refundable"
 
 
 c07100 = variable_alias("c07100", income_tax_non_refundable_credits)

--- a/policyengine_us/variables/gov/irs/credits/income_tax_refundable_credits.py
+++ b/policyengine_us/variables/gov/irs/credits/income_tax_refundable_credits.py
@@ -7,13 +7,4 @@ class income_tax_refundable_credits(Variable):
     definition_period = YEAR
     label = "federal refundable tax credits"
     unit = USD
-    adds = [
-        "eitc",
-        "refundable_american_opportunity_credit",
-        "refundable_ctc",
-        "recovery_rebate_credit",
-        "refundable_payroll_tax_credit",
-        "premium_tax_credit",
-        "ecpa_filer_credit",
-        "ecpa_adult_dependent_credit",
-    ]
+    adds = "gov.irs.credits.refundable"

--- a/setup.py
+++ b/setup.py
@@ -36,7 +36,7 @@ setup(
         "microdf_python==0.3.0",
         "pandas==1.5.2",
         "pathlib==1.0.1",
-        "policyengine-core==1.10.24",
+        "policyengine-core==1.11.0",
         "pytest==5.4.3",
         "pytest-dependency==0.5.1",
         "pyyaml==6.0",

